specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getComputeOneSkuUsages.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
+    "resourceName": "standardNDSFamily",
+    "api-version": "2021-03-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "limit": 10,
+          "unit": "Count",
+          "name": {
+            "value": "standardNDSFamily",
+            "localizedValue": "Standard NDS Family vCPUs"
+          },
+          "isQuotaApplicable": true
+        }
```

Comment:
If Microsoft.Quota/quotaLimits represents manageable resources, it should return the top level properties of ARM resources like type, name, and ID.

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource)

In this example,
{
  "type": "Microsoft.Quota/quotaLimits",
  "name: "standardNDSFamily",
  "id": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimits/standardNDSFamily",
  "properties": {
    "limit": 10,
...
  }
}

Reply:
Ok, I will add it.

Reply:
Added id field, as suggested.

Reply:
Please add 'type', 'name' id' at the top level not inside "properties".